src/DataCollection.UWP/Views/IdentifiedFeaturePopup.xaml

```diff
@@ -1,15 +1,121 @@
+                                                Grid.Column="1" Grid.Row="1" HorizontalAlignment="Right" Margin="0,0,10,0" 
+                                                Command="{Binding ElementName=IdentifiedFeatureUserControl, Mode=OneWay, Path=DataContext.IdentifiedFeatureViewModel.SetSelectedDestinationRelationshipCommand}" CommandParameter="{Binding}"/>
+                                    </Grid>
+                                </DataTemplate>
+                            </ItemsControl.ItemTemplate>
+                        </ItemsControl>
+
+                        <!--List of relationships where identified feature is origin-->
+                        <ItemsControl ItemsSource="{x:Bind IdentifiedFeatureViewModel.OriginRelationships, Mode=OneWay}" HorizontalContentAlignment="Stretch" Grid.Row="2" Margin="5">
+                            <ItemsControl.ItemTemplate>
+                                <DataTemplate>
+                                    <StackPanel Orientation="Vertical">
+                                        <Grid>
+                                            <Grid.ColumnDefinitions>
+                                                <ColumnDefinition Width="*"/>
+                                                <ColumnDefinition Width="40"/>
+                                            </Grid.ColumnDefinitions>
+                                            <TextBlock Text="{Binding RelatedTable.TableName, Mode=OneWay}" FontSize="17" Padding="10" Grid.Column="0"/>
+                                        </Grid>
+                                        <ItemsControl ItemsSource="{Binding OriginRelatedRecords,Mode=OneWay}" BorderThickness="0" HorizontalContentAlignment="Stretch" Margin="3">
+                                            <ItemsControl.ItemTemplate>
+                                                <DataTemplate>
+                                                    <Grid>
+                                                        <Grid.ColumnDefinitions>
+                                                            <ColumnDefinition Width="*"/>
+                                                            <ColumnDefinition Width="40"/>
+                                                        </Grid.ColumnDefinitions>
+                                                        <Grid.RowDefinitions>
+                                                            <RowDefinition Height="5"/>
+                                                            <RowDefinition Height="*"/>
+                                                        </Grid.RowDefinitions>
+                                                        <StackPanel Orientation="Vertical" Grid.Column="0" Grid.Row="1" x:Name="OriginRecordGrid"> 
+                                                            <TextBlock Text="{Binding DisplayedFields[0].Field.Label, Mode=OneWay}" FontSize="12" FontWeight="Bold" Margin="5,0,0,0"/>
```

Comment:
This is just a personal opinion, but I always thought Bold felt too heavy on UWP. Have you thought about Semibold? It looks like the type ramp uses that as the intermediate step just above 'body': https://docs.microsoft.com/en-us/windows/uwp/design/style/typography#type-ramp 

I recommend using a built-in text-style if possible, or defining a custom style to keep things consistent.

Reply:
Yes, that definitely doesn't match. It's leftover from WPF. Thanks for the resource!